pkg/datapath/loader/netlink.go

```diff
@@ -93,8 +94,12 @@ func replaceDatapath(ctx context.Context, ifName, objPath, progSec, progDirectio
 			"obj", objPath, "sec", progSec}
 	} else {
 		loaderProg = "tc"
+		var tcPrio string
+
+		tcPrio = strconv.Itoa(option.Config.TCFilterPriority)
+		log.Infof("tc filter using priority %s for interface %s", tcPrio, ifName)
```

Comment:
I think this should be a debug message. The value is coming directly from the flag that is already logged and we also have the priority of tc BPF programs in the bugtool reports.

Reply:
Agree